tools/cdk-release/lib/lifecycles/changelog.ts

```diff
@@ -50,9 +92,8 @@ export async function changelog(
   const presetConfig = await conventionalChangelogPresetLoader({
     name: 'conventional-changelog-conventionalcommits',
   });
-  debugObject(args, 'conventionalChangelogPresetLoader returned', presetConfig);
```

Comment:
Because it's an exceptionally verbose piece of output (dozens of lines) that starkly contrasts with the rest of the reasonably minimal input. I can see it being useful when developing this function, but in a "normal" verbose run it got in the way. Future work to maybe define an "Info" vs "Debug" level, or maybe "-v" vs "-vv", but it didn't seem worth the effort here.